main.js

```diff
@@ -0,0 +1,120 @@
+const API_KEY = "6d29bbab";
+
+document.getElementById("form-submit").addEventListener("submit", getIp);
+let page = 1;
+let searchWord = document.getElementById("searchWord");
+function getIp(e) {
+  e.preventDefault();
+  let iVal = searchWord.value;
```

Comment:
It should have more meaningful name
```suggestion
  const iVal = searchWord.value;
```